ci.jsonnet

```diff
@@ -237,7 +237,7 @@ local part_definitions = {
         # Workaround for NFI when building with different Truffle versions
         ["mx", "clean"],
         ["mx", "build"],
-        ["mx", "fetch-languages", "--language:llvm", "--language:ruby"],
+        ["mx", "fetch-languages", "--language:ruby"],
```

Comment:
Why was `--language:llvm` removed?

Reply:
So this is what causes all the SVM benchmarks and things to fail? But why fetch `llvm` if it isn't then used in the next line? How does this work?

Reply:
Yes. `mx fetch-languages` doesn't understand suite dependencies so we need to make sure to fetch llvm first then ruby so references to sulong in the truffleruby suite can resolve properly.